docroot/sites/all/modules/features/ilr_forms_students/ilr_forms_students.features.inc

```diff
@@ -21,6 +21,57 @@ function ilr_forms_students_ctools_plugin_api($module = NULL, $api = NULL) {
  */
 function ilr_forms_students_default_entityform_type() {
   $items = array();
+  $items['alumni_assoc_board_interest'] = entity_import('entityform_type', '{
+    "type" : "alumni_assoc_board_interest",
+    "label" : "Alumni Association Board Interest",
+    "data" : {
+      "draftable" : 0,
+      "draft_redirect_path" : "",
+      "draft_button_text" : "",
+      "draft_save_text" : { "value" : "", "format" : "full_html" },
+      "submit_button_text" : "",
+      "submit_confirm_msg" : "",
+      "your_submissions" : "",
+      "disallow_resubmit_msg" : "",
+      "delete_confirm_msg" : "",
+      "page_title_view" : "",
+      "preview_page" : 0,
+      "submission_page_title" : "Thank you",
+      "submission_text" : {
+        "value" : "\\u003Cp\\u003EThank you for your interest in running for the ILR Alumni Association Board.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EThe next step is to fill out the \\u003Ca href=\\u0022https:\\/\\/www.ilr.cornell.edu\\/sites\\/ilr.cornell.edu\\/files\\/ILRAANominationForm.docx\\u0022\\u003Efull nomination form\\u003C\\/a\\u003E. You will receive a confirmation email shortly, which will also include this link.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EIf you have any questions, please contact Sue Sappington, \\u003Ca href=\\u0022mailto:ss446@cornell.edu\\u0022\\u003Ess446@cornell.edu\\u003C\\/a\\u003E.\\u003C\\/p\\u003E\\r\\n",
+        "format" : "full_html"
+      },
+      "submission_show_submitted" : 1,
+      "submissions_view" : "default",
+      "user_submissions_view" : "default",
+      "form_status" : "ENTITYFORM_OPEN",
+      "roles" : {
+        "1" : "1",
+        "2" : "2",
+        "11" : 0,
+        "16" : 0,
+        "41" : 0,
+        "51" : 0,
+        "46" : 0,
+        "21" : 0,
+        "26" : 0,
+        "31" : 0,
+        "56" : 0,
+        "61" : 0
+      },
+      "resubmit_action" : "new",
+      "redirect_path" : "",
+      "instruction_pre" : { "value" : "", "format" : "full_html" },
+      "send_confirmation_email_to_user" : 1,
+      "send_confirmation_email_to_admin" : 1,
+      "add_user_name_to_subject_line" : 1,
+      "kerberized" : 0,
+      "add_payment_processing" : 0,
+      "form_submission_ilr_recipient" : "cgh2@cornell.edu"
```

Comment:
Thanks for noticing that my name was in there as the recipient.  I'll change it to Sue.  For the name of the file, Nathan didn't think you could create a redirect to an uploaded file, so I had Sue change the name to what's in the email now, so that it could be used every year.  The original filename had the year in it.